src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStreamPacketType.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.Data.SqlClientX.IO
+{
+    internal enum TdsStreamPacketType
+    {
+        PreLogin = 0x12,
+        Login7 = 0x10,
+        SQLBatch = 0x01,
```

Comment:
I think we should be adopting the C# naming standard for this project, following `PascalCase` even for acronyms.

Reply:
I agree. I didn't pay attention to the naming. Thanks for calling this out.